src/modules/remote.json

```diff
@@ -0,0 +1,157 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "Remote",
+    "description": "A module for managing Remote keys, paring and programming.",
+    "version": "0.0.0"
+  },
+  "methods": [
+    {
+      "name": "state",
+      "summary": "Read-only property method for the remote control state. This is a getter for the state or a way to subscribe to changes to the state.",
+      "params": [],
+      "result": {
+        "name": "remoteList",
+        "summary": "Contains a list of remotes controls paired to the device.",
+        "schema": {
+          "$ref": "#/components/schemas/RemoteControlState"
+        }
+      },
+      "examples": [
+        {
+          "name": "Remote State Example",
+          "params": [],
+          "result": {
+            "remoteList": [
+              {
+                "model": "PR2",
+                "batteryPercentage": "70"
```

Comment:
I think having the battery percent in a subscribable state might not work. Won't the app get an event after every percent of battery used?
Or were you intending the subscribe event would only fire when a new remote is added or removed? If that is the case, I would maybe change the method name to like `list` or `remotes`

Reply:
Or this method will only return the list of paired remotes right? Maybe that needs to be conveyed as well.
```
let list = await Remote.paired()
```
```
Remote.paired((list) => {
  // list of paired remotes
})
```

Reply:
This is a get method and not subscribable, just takes the snapshots of the list of remotes connected to the device and their current battery levels.

Reply:
Updated the method signature to make it more meaningful